app/views/questionnaire/components/universityDegreeCategory.scala.html

```diff
@@ -0,0 +1,19 @@
+@(questionnaireForm: Form[_root_.forms.EducationQuestionnaireForm.Data])(implicit lang: play.api.i18n.Lang)
+
+@import views.html.questionnaire.components.universityDegreeCategoryDetails
+@import views.html.widgets.{ checkbox, selectValueDifferentFromDisplayedText }
+
+<section>
+    @selectValueDifferentFromDisplayedText(questionnaireForm("universityDegreeCategory"),
+        models.view.questionnaire.UniversityDegreeCategory.seq.toList)(
+        "-- Select one --") {
+        @Messages("universityDegreeCategory.question")
+    }
+    <div class="form-group">
+    @checkbox(questionnaireForm("preferNotSay_universityDegreeCategory"), Some("universityDegreeCategory")) {
```

Comment:
Is this second param Some("universityDegreeCategory") needed, if so what is it for?